examples/cpu-profiler/test.sh

```diff
@@ -7,11 +7,13 @@ sleep 1
 
 curl -v http://localhost:3000/start-profiling
 
-node node_modules/.bin/autocannon -l -R 200000 -c 2000 -d 20 http://localhost:3000
+node --cpu-prof --cpu-prof-name=samples-cpu-prof.cpuprofile \
```

Comment:
If you mean profiling `node index.js`, I tried that but couldn't get its `.cpuprofile` to be generated reliably, possibly because of the `kill -KILL`. After trying `kill -TERM`, `kill -INT` and just plain `kill` but still no joy, I settled for autocannon because its `.cpuprofile` isn't much bigger than the current `samples.cpuprofile` produced by `index.js`.

(I also tried profiling 0x ... its `.cpuprofile` is about 10 times bigger than the one for autocannon)

Reply:
I would add another cpu-profiler test instead of incrementing this one. Something like `examples/cpu-profiler/cpu-prof-test.sh` and testing a HTTP application should be enough

Reply:
Ok 7698b50084b2b5088f181aee4620ad6351f08cd7